pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -321,6 +324,7 @@ private static long getRandomInitialDelayInSeconds() {
       AutoRebalanceStrategy.class.getName();
   private static final int DEFAULT_LEAD_CONTROLLER_RESOURCE_REBALANCE_DELAY_MS = 300_000; // 5 minutes
   private static final String DEFAULT_DIM_TABLE_MAX_SIZE = "200M";
+  private static final int EMPTY_THREAD_POOL = 0;
```

Comment:
Suggest using `-1` to represent unspecified. Default to empty thread pool can be a little bit confusing